com.dreamsourcelab.DSView.metainfo.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.dreamsourcelab.DSView</id>
+
+  <name>DSView</name>
+  <summary>An open source multi-function instrument for everyone </summary>
+
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-only</project_license>
+  <content_rating type="oars-1.0" />
+
+  <description>
+    <p>
+      NOTE: This wrapper is not verified by, affiliated with, or supported by DreamSourceLab.
+    </p>
+    <p>
+      DSView is a GUI program for supporting various instruments from DreamSourceLab, including logic analyzers, oscilloscopes, etc. DSView is based on the sigrok project.
+    </p>
+    <p>
+      The sigrok project aims at creating a portable, cross-platform, Free/Libre/Open-Source signal analysis software suite that supports various device types (such as logic analyzers, oscilloscopes, multimeters, and more).
+    </p>
+    <p>
+      Adding following line to /etc/udev/rules.d/60-dreamsourcelab.rules
+    </p>
+    <p>
+      SUBSYSTEM=="usb", ATTRS{idVendor}=="2a0e", MODE="0666"
+    </p>
+  </description>
+
+  <releases>
+    <release version="1.3.0" date="2023-07-18"/>
+  </releases>
+
+  <launchable type="desktop-id">DSView.desktop</launchable>
```

Comment:
this doesn't match either the name in `rename-desktop-file` nor the proper name it should have.
While `rename-desktop-file` likely doesn't care, since this file should be submitted upstream it's a good idea to be correct.